assets/tex/pg.sty

```diff
@@ -0,0 +1,4 @@
+\ProvidesFile{pg}
```

Comment:
Shouldn't this be `\ProvidesPackage{pg}`?  It is my understanding that this should be used for a `.sty` file, `\ProvidesClass` for a `.cls` file, and `\ProvidesFile` for anything else.

The same is true of the `webwork2.sty` file now in the webwork2 code base.  I missed this when you added that file.

The `\ProvidesFile` command should work fine for these, but the `\ProvidesPackage` is probably more correct, and it does do some checking specifically for packages as I understand it.  It also has an optional package version argument.

Reply:
I'm not an expert with this, but I think because it is not a true package, and it's rather more like locally defined macros, that `ProvidesFile` was the recommendation. But I was just reading a stackexchange post, not authoritative.

Reply:
The stack exchange post that I found says the opposite of the one that you see.

I don't believe that LaTeX distinguishes between local and system packages, or that this is not a true package.